File: 0001 Code Chef/0022 Fitness/implementation/0011 lua/fitness.lua
test = tonumber(io.read())
while(test > 0) do
    distance = tonumber(io.read())
    distance_walk_per_day = distance * 2
    distance_walk_per_week = distance_walk_per_day * 5
    print(distance_walk_per_week)
    test = test - 1
end

File: 0001 Code Chef/0022 Fitness/implementation/0009 dart/fitness.dart
import 'dart:io';

void main(){
  String? stringInput = stdin.readLineSync();
  if(stringInput != null){
    int? test = int.tryParse(stringInput);
    if(test != null){
      int count = test;
      while(count > 0){
        String? stringInput = stdin.readLineSync();
        if(stringInput != null){
          int? distance = int.tryParse(stringInput);
          if(distance != null){
            int distance_walk_per_day = 2 * distance;
            int distance_walk_per_week = 5 * distance_walk_per_day;
            print(distance_walk_per_week);
          }
        }
        count = count - 1;
      }
      }
    } 
  }